scripts/travis/apt-install

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e -x
+
+export DEBIAN_FRONTEND=noninteractive
+
+install_retry_counter=0
+max_apt_retries=5
```

Comment:
It looks like `apt-get` has support for `APT::Acquire::Retries`, which allows to set
> Number of retries to perform. If this is non-zero APT will retry failed files the given number of times. 

Additionally, we could set a few other options as well, for example
```sh
$ cat scripts/travis/80-apt-conf 
APT::Get::Install-Recommends "false";
APT::Get::Install-Suggests "false";
APT::Get::Assume-Yes "true";
APT::Acquire::Retries 5;
```

```
COPY scripts/travis/80-apt-conf /etc/apt/apt.conf.d/
```

Reply:
Interesting. Would be good to know which way, my loop or the apt settings, performs more reliable in Travis. Probably hard to tell. 

Any other comments/ideas which approach is better suited for us.

Reply:
I haven't thoroughly evaluated the robustness of the `Retries` option but the [implementation](https://github.com/Debian/apt/blob/289ee74dd23cba7e08b08c6c3602bcf4bf8167bc/apt-pkg/acquire-worker.cc#L610) and [test](https://github.com/Debian/apt/blob/289ee74dd23cba7e08b08c6c3602bcf4bf8167bc/test/integration/test-bug-869859-retry-downloads#L24)  in apt-get look reasonable to me. I guess the advantage of using the built-in functionality may be an improved performance as on failure apt-get will retry to connect rather than restating the whole process.

Reply:
I would really prefer to use the built-in method instead of my shell script around apt-get. I expect, however, that the current shell loop based variant is more robust. I also found https://travis-ci.community/t/apache-arrow-sporadic-build-failures-due-to-hash-sum-mismatch-errors/2563 which seems to be an error like we also have seen. Especially with the sleep in between tries we might have a better chance to recover from a network error.

But using a built-in solution instead of a shell script would be nice. Not sure what is better.